### PR TITLE
Add 2 questions to foundations

### PR DESCRIPTION
**Pull request by Andresmup**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

## What's new?

Add two questions (104, 105) to Foundations.
## Related issue(s)

#90

### CODE REVIEW

File: content/questions/foundations/question-104.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: "Question 104"
+question: "If a GitHub Discussion is converted into an Issue and the issue is closed or referred to using its corresponding # number, will the discussion be modified?"
+draft: false
+---
+
+
+> https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion
+1. [ ] Yes
+1. [x] No
+> When you "convert" a discussion into an Issue. This is a new issue, just linked by a soft link to the discussion and prepopulate the issue info within. There is no real connection or relationship.
```
Comment (FidelusAleksander) [inline content/questions/foundations/question-104.md:12]:
```suggestion
> When you "convert" a discussion into an Issue it essentially creates a new issue that is soft linked to the discussion and pre-fills the issue information. There isn't a bigger connection between them.
```

File: content/questions/foundations/question-105.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: "Question 105"
+question: "Which information can be found in the Pulse section in the Insights tab of a repository? (Choose three)"
+draft: false
+---
+
+
+
+
+> https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/using-pulse-to-view-a-summary-of-repository-activity
+- [x] Pull requests open/close ratio
```
Comment (FidelusAleksander) [inline content/questions/foundations/question-105.md:12]:
```suggestion
- [x] Pull requests open/merged ratio
```

File: content/questions/foundations/question-105.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- [x] Summary of activity
```
Comment (FidelusAleksander) [inline content/questions/foundations/question-105.md:13]:
```suggestion
- [x] Summary of repository activity
```

File: content/questions/foundations/question-105.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+- [ ] Ammount of additions and deletions of code
```
Comment (FidelusAleksander) [inline content/questions/foundations/question-105.md:14]:
```suggestion
- [ ] Amount of code line additions and deletions
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Thanks for contributing again @Andresmup !

I've left small suggestions/fixes. Please re-request review once addressed :heavy_check_mark:

**Review by Andresmup — COMMENTED** [review]
Commit changes to update questions

File: content/questions/foundations/question-104.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+title: "Question 104"
+question: "If a GitHub Discussion is converted into an Issue and the issue is closed or referred to using its corresponding # number, will the discussion be modified?"
+draft: false
+---
+
+
+> https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion
+1. [ ] Yes
+1. [x] No
+> When you "convert" a discussion into an Issue it essentially creates a new issue that is soft linked to the discussion and pre-fills the issue information. There isn't a bigger connection between them.
```
Comment (FidelusAleksander) [inline content/questions/foundations/question-104.md:12]: Sorry - missed that one previously.

The explanations are shown only for wrong answers - at the end of the practice test. There is no use for explanations for correct answer - they will not be shown
```suggestion
1. [x] No
1. [ ] Yes
> When you "convert" a discussion into an Issue it essentially creates a new issue that is soft linked to the discussion and pre-fills the issue information. There isn't a bigger connection between them.
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Last thing and we are good to merge

**Comment by Andresmup** [comment]
@FidelusAleksander done 😃

**Review by FidelusAleksander — APPROVED** [review]
Thanks! I will get this merged

**Pull request merged**